script/Ciel/Day 13/13_01_01_CIEL13_5.txt

```diff
@@ -470,7 +470,7 @@ Everything about you is a lie!
 [sha:7d71c55f207c7116e10c2d5a429355a33e170d21]{
 -- Page 545, Offset 34835.
 -- 降りそそぐ罵倒は呪いのようだ。
-She hurls abuse at Senpai as though cursing her.
+Noel's barrage of abuse is almost curse-like.
```

Comment:
Hmm don't love this. I feel like it needs more structure in EN to make the comparison sound sane. 
> Noel's stream of abuse is so constant it almost sounds like she's chanting a curse.

Reply:
Seems a bit long. Could kinda change it like this if you want one more qualifier, but it seems fine to me imo

>Noel's barrage of abuse is almost curse-like in fervor.

>Noel's barrage of abuse is almost curse-like in intensity.

Reply:
I think my main issue is that 'curse-like' is an odd descriptor in EN. Like, what is that supposed to call to mind? I guess he didn't exactly expound on it in JP either, so unless there's a more intrinsic association (curses are long litanies, etc) in JP that isn't being conveyed here I guess eh what you have is fine

Reply:
I think the association is just that she's so vile it's almost like her words aren't just, well, a bunch of words, but trying to actually hurt Ciel.

I remember now why I changed this, it's because "cursing" someone in English has the double-meaning of just well, cursing at them. So it sounds weirdly redundant. "curse-like" avoided that problem, though I think it can be avoided in other ways too.